src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
-body {
+$strong-color: #060B35;
+$light-color: #616070;
+$blue-color: #00ACDC;
+$white-color: #fff;
```

Comment:
Will be better if you move variables to the variables file

Reply:
In my browser everything is working. Regarding stars. There is a problem with npm.

Reply:
If u want to check file u can copy html and css file and run live server. I cannot fix problem with npm.